test/meson.build

```diff
@@ -0,0 +1,21 @@
+gtest_dep = dependency('gtest', main : true, required : false)
```

Comment:
The `required` parameter should be set to `false` to prevent the build from proceeding if the GTest library is not found in the build environment. It should only be set to `true` when a wrap file is explicitly defined.

Reply:
ok understood.. Addressed